tests/acceptance/features/bootstrap/TrashbinContext.php

```diff
@@ -671,6 +671,28 @@ public function deleteFileFromTrashbin(string $user, string $originalPath):void
 		$this->featureContext->pushToLastStatusCodesArrays();
 	}
 
+	/**
+	 * @Given /^user "([^"]*)" has deleted the (?:file|folder|entry) with original path "([^"]*)" from the trashbin$/
+	 *
+	 * @param string $user
+	 * @param string $originalPath
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function userHasDeletedTheFolderWithOriginalPathFromTheTrashbinUsingTheTrashbinApi(string $user, string $originalPath) :void {
```

Comment:
Please, follow the changes you have made to the related lines as well.
```suggestion
	public function userHasDeletedTheFolderWithOriginalPathFromTheTrashbin(string $user, string $originalPath) :void {
```